spec/models/container_group_spec.rb

```diff
@@ -49,7 +49,7 @@
     it "preloads the conditions" do
       condition_other
       cr = condition_ready
-      cg = ContainerGroup.includes(:ready_condition_status).references(:ready_condition_status => {}).find_by(:id => container_group.id)
+      cg = ContainerGroup.includes(:ready_condition_status).references(:ready_condition_status).find_by(:id => container_group.id)
```

Comment:
is the `ready_condition_status` correct?  It feels odd that references takes an array of table_names and not relations but we have a virtual column here that doesn't follow that mold.

Reply:
sorry, specifically, `.references(:ready_condition_status)`

Reply:
@jrafanie it is wonky, but correct, as defining a `virtual_attribute`/`virtual_column` sets a `include` so it can be "included in SQL" (as @kbrock would put it):

https://github.com/ManageIQ/activerecord-virtual_attributes/blob/3aefda89837a945e782a8f3b936d2ec543b202fa/lib/active_record/virtual_attributes/virtual_includes.rb#L5-L6

Reply:
Ok, thanks!

Reply:
I hate `references(:field)` but @Fryguy loves it. sigh

It should probably just be an includes

Reply:
> It should probably just be an includes

I think you made the right call by not changing it here and just modifying the arguments passed.  This PR was just meant for truing up our usage of `.references` to make it #Correct™, not starting to re-work existing code to have it functionally work differently.